### ISSUE TITLE
Contact form accepts bad input but won't show errors

### ISSUE THREAD

**Issue by orsi** [danispinxo/jesse-site#4]
The filling out the contact form, it looks like bad emails or phone numbers will cause the form to error on submitting, but there is no user feedback to show. Ideally, email inputs should at least warn that you should input a valid email. For phone numbers, it may be alright to just submit as is, as formatting/checking phone number formats are difficult.

<img width="1344" alt="Screen Shot 2022-11-16 at 10 22 35 PM" src="https://user-images.githubusercontent.com/7156655/202347215-03a2b9de-969e-48aa-9fb4-9afb43e93e38.png">

**Comment by danispinxo** [comment]
Good catch, QA tester! 😉 I've added clearer form validation/error messages on the email input so the user knows when a form could not be submitted due to invalid input.
